libpod/container_attach.go

```diff
@@ -105,7 +93,7 @@ func (c *Container) attachContainerSocket(resize <-chan remotecommand.TerminalSi
 	}
 
 	// Put both input and output into raw
```

Comment:
Nit: can you update the comment to note that this only happens if we have a terminal?

Reply:
done